standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreServerUtils.java

```diff
@@ -122,7 +123,8 @@ public class MetaStoreServerUtils {
   private static final Logger LOG = LoggerFactory.getLogger(MetaStoreServerUtils.class);
 
   public static final String JUNIT_DATABASE_PREFIX = "junit_metastore_db";
-
+  public static final String TABLE_TYPE_PROP = "table_type";
+  public static final String ICEBERG_TABLE_TYPE_VALUE = "iceberg";
```

Comment:
no need to define again, they are defined here, can use the same
```
import static org.apache.hadoop.hive.metastore.HiveMetaHook.ICEBERG;
import static org.apache.hadoop.hive.metastore.HiveMetaHook.TABLE_TYPE;
```

Reply:
Done